### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
     </a>
-    <a href="https://bulma.io/" target="_blank" rel="external">
-        <img src="docs/images/bulma-logo.png" height="100px">
+    <a href="https://bulma.io/" target="_blank">
+        <img src="https://bulma.io/assets/brand/Bulma%20Logo.svg" height="80px" alt="Bulma">
```
Comment (vjik) [inline README.md]: It looks strange: 

image

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (luizcmarin) [reply README.md]: Yes! :))
Choose one https://bulma.io/brand/

Comment (vjik) [reply README.md]: Need to choose image that will look good in both dark and light theme. Also need to add this image to repo.

Comment (luizcmarin) [reply README.md]: removed

File: README.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
-    </a>
-    <a href="https://bulma.io/" target="_blank" rel="external">
-        <img src="docs/images/bulma-logo.png" height="100px">
```
Comment (vjik) [inline README.md:6]: Fix image path and keep this image.

**Review by vjik — CHANGES_REQUESTED** [review]

**Pull request closed without merging**